### ISSUE TITLE
rust 1.80.0 build issue

### ISSUE THREAD

**Issue by chenrui333** [qarmin/czkawka#1341]
time crate needs to be updated to 0.3.36

see https://github.com/time-rs/time/issues/693

related error log:

```
  error[E0282]: type annotations needed for `Box<_>`
    --> /Users/brew/Library/Caches/Homebrew/cargo_cache/registry/src/index.crates.io-6f17d22bba15001f/time-0.3.34/src/format_description/parse/mod.rs:83:9
     |
  83 |     let items = format_items
     |         ^^^^^
  ...
  86 |     Ok(items.into())
     |              ---- type must be known at this point
     |
     = note: this is an inference error on crate `time` caused by an API change in Rust 1.80.0; update `time` to version `>=0.3.35` by calling `cargo update`
  
  For more information about this error, try `rustc --explain E0282`.
```

**Comment by vlad-sundunchan** [comment]
If you encounter the error type annotations needed for Box<_> when installing the Anchor CLI, try changing your Rust version to 1.79.0 and attempt the installation again.

`rustup default 1.79.0`

**Comment by qarmin** [comment]
Fixed by #1354
